gateway/src/main/java/io/zeebe/gateway/impl/broker/request/BrokerRequest.java

```diff
@@ -63,6 +65,10 @@ public void serializeValue() {
 
   protected abstract T toResponseDto(DirectBuffer buffer);
 
+  public String getType() {
+    return COMMON_WORDS.matcher(this.getClass().getSimpleName()).replaceAll("");
```

Comment:
Can we use the valueType and intent to build the request type instead of parsing the class name?

Reply:
The BrokerRequest doesn't have access to the valueType and intent. This was a hack around it. It's not pretty but it works for all the request types since they all follow the same format

Reply:
Why not add it in `BrokerExecuteCommand`? All requests extends it, right? Anyway, it is just a suggestion.